snippets/language-svg.cson

```diff
@@ -1,101 +1,352 @@
+    'descriptionMoreURL': 'https://developer.mozilla.org/en-US/docs/Web/SVG/Element/feComposite'
+  'feDiffuseLighting':
+    'prefix': 'feDiffuseLighting'
+    'body': '<feDiffuseLighting in="${1:SourceGraphic}" result="${2:light}" lighting-color="${3:white}"><fePointLight x="$4" y="$5" z="$6" /></feDiffuseLighting>'
+    'description': 'This filter primitive lights an image using the alpha channel as a bump map. The resulting image, which is an RGBA opaque image, depends on the light color, light position and surface geometry of the input bump map. The light map produced by this filter primitive can be combined with a texture image using the multiply term of the arithmetic operator of the <fecomposite> filter primitive. Multiple light sources can be simulated by adding several of these light maps together before applying it to the texture image.'
+    'descriptionMoreURL': 'https://developer.mozilla.org/en-US/docs/Web/SVG/Element/feDiffuseLighting'
+  'feDistantLight':
+    'prefix': 'feDistantLight'
+    'body': '<feDistantLight azimuth="${1:240}" elevation="${2:20}"/>'
+    'description': 'This filter primitive define a distant light source that can be used within a lighting filter primitive : <fediffuselighting> or <fespecularlighting>.'
+    'descriptionMoreURL': 'https://developer.mozilla.org/en-US/docs/Web/SVG/Element/feDistantLight'
+  'feGaussianBlur':
+    'prefix': 'feGaussianBlur'
+    'body': '<feGaussianBlur in="${1:SourceGraphic}" stdDeviation="${1:5}" />'
+    'description': 'The filter blurs the input image by the amount specified in stdDeviation, which defines the bell-curve.'
+    'descriptionMoreURL': 'https://developer.mozilla.org/en-US/docs/Web/SVG/Element/feGaussianBlur'
+  'fePointLight':
+    'prefix': 'fePointLight'
+    'body': '<fePointLight x="$1" y="$2" z="$3" />'
+    'description': ''
+    'descriptionMoreURL': 'https://developer.mozilla.org/en-US/docs/Web/SVG/Element/fePointLight'
+  'feSpotLight':
+    'prefix': 'feSpotLight'
+    'body': '<feSpotLight x="${1:360}" y="${2:5}" z="${3:30}" limitingConeAngle="${4:20}" pointsAtX="${5:390}" pointsAtY="${6:80}" pointsAtZ="${7:0}"/>'
+    'description': 'The feSpotLight element is one of the ligth source elements used for SVG files.'
+    'descriptionMoreURL': 'https://developer.mozilla.org/en-US/docs/Web/SVG/Element/feSpotLight'
+  'filter':
+    'prefix': 'filter'
+    'body': '<filter id="$1" x="${2:0}" y="${3:0}">$4</filter>'
+    'description': 'The filter element serves as container for atomic filter operations. It is never rendered directly. A filter is referenced by using the filter attribute on the target SVG element.'
+    'descriptionMoreURL': 'https://developer.mozilla.org/en-US/docs/Web/SVG/Element/filter'
+  'group':
+    'prefix': 'g'
+    'body': '<g $1>$2</g>'
```

Comment:
That is placeholder for attributes, that the developer maybe wants to add like for example:
```
  <g id="group1" fill="red">
  
  </g>
```